ruby/red-arrow/ext/arrow/raw-records.cpp

```diff
@@ -305,9 +305,10 @@ namespace red_arrow {
   }
 
   VALUE
-  record_batch_each_raw_record(VALUE rb_record_batch){
+  record_batch_each_raw_record(VALUE rb_record_batch) {
     auto garrow_record_batch = GARROW_RECORD_BATCH(RVAL2GOBJ(rb_record_batch));
     auto record_batch = garrow_record_batch_get_raw(garrow_record_batch).get();
+    RETURN_SIZED_ENUMERATOR(rb_record_batch, 0, 0, record_batch->num_rows());
```

Comment:
We should use `nullptr` for a null pointer in C++:

```suggestion
    RETURN_SIZED_ENUMERATOR(rb_record_batch, 0, nullptr, record_batch->num_rows());
```

Reply:
Thank you so much! 
fix: [3563046](https://github.com/apache/arrow/pull/37600/commits/3563046fe239850350ad334908d0d79be641c94c)